src/routes/usersRoutes.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const usersController = require('../controllers/usersController');
+
+router.get('/users', usersController.getUsers);
+router.get('/users/:id', usersController.getUserById);
+router.post('/users', usersController.createUser);
+router.patch('/users/:id', usersController.updateUser);
+router.delete('/users/:id', usersController.deleteUser);
```

Comment:
if you'd apply my suggestion in `createServer.js` you could omit typing `/users` 5 times 😄